.travis.yml

```diff
@@ -19,6 +19,9 @@ install:
   - pip install testfixtures
   - pip install unittest2
   - pip install scikit-learn
+  - pip install theano
+  - pip install tensorflow
+  - pip install keras
```

Comment:
You are right @chinmayapancholi13, add this to [this list](https://github.com/RaRe-Technologies/gensim/blob/develop/setup.py#L229)